picard/ui/item.py

```diff
@@ -99,6 +100,19 @@ def error_append(self, msg):
     def clear_errors(self):
         self._errors = []
 
+    def _cover_art_description(self, images, detailed=True):
+        # Return the number of cover image found in metadata.images. Used I the cover column of the UI.
```

Comment:
To describe the method, use Python docstring (see https://www.python.org/dev/peps/pep-0257/) -> replace `# comment` with `""" comment """` (it can be multi-line).

What does `Used I the cover column of the UI.` mean? (tu peux expliquer en français)

Reply:
Maybe something like this for a full function description:

```python
"""Return the number of cover art images in `images` for display in the UI

Args:
    images: An instance of picard.util.imagelist.ImageList
    detailed: If True, return a detailed text about the images and whether they are the same across all tracks.
              If False only return the number as a string.

Returns:
    A string explaining the cover art image count.
"""
```

Apart from this description this PR looks good to me. Good work!